biopet-tools/src/main/scala/nl/lumc/sasc/biopet/tools/ReplaceContigsGtfFile.scala

```diff
@@ -0,0 +1,85 @@
+package nl.lumc.sasc.biopet.tools
+
+import java.io.{File, PrintWriter}
+
+import nl.lumc.sasc.biopet.utils.ToolCommand
+import nl.lumc.sasc.biopet.utils.annotation.Feature
+
+import scala.io.Source
+
+/**
+  * Created by pjvan_thof on 30-5-17.
+  */
+object ReplaceContigsGtfFile extends ToolCommand {
+  case class Args(input: File = null,
+                  output: File = null,
+                  contigs: Map[String, String] = Map(),
+                  writeAsGff: Boolean = false)
+      extends AbstractArgs
+
+  class OptParser extends AbstractOptParser {
+    opt[File]('I', "input") required () valueName "<file>" action { (x, c) =>
+      c.copy(input = x)
```

Comment:
Waarom grote letter `I` voor input en kleine letter `o` voor output? Zijn hier standaarden voor in biopet?